ladybug/src/main/java/nl/nn/ibistesttool/Debugger.java

```diff
@@ -177,7 +183,7 @@ public Object preserveInput(String correlationId, Object input) {
 		return testTool.outputpoint(correlationId, null, "PreserveInput", input);
 	}
 	
-	public String rerun(String correlationId, Report originalReport, SecurityContext securityContext, ReportRunner reportRunner) {
+	public String rerun(String correlationId, Report originalReport, SecurityContext securityContext) {
```

Comment:
Hier wordt de interface van deze method aangepast, terwijl dat bij de aanroepende partij niet gebeurd.
Het lijkt mij dat deze aanpassing niet in deze pull request had moeten zitten.

Reply:
Zie reactie over getMessageWithResolvedVariables(reportRunner).